scriptworker/test/__init__.py

```diff
@@ -261,7 +261,7 @@ def tmpdir2():
 def rw_context():
     with tempfile.TemporaryDirectory() as tmp:
         context = Context()
-        context.config = dict(deepcopy(DEFAULT_CONFIG))
+        context.config = unfreeze_values(DEFAULT_CONFIG)
```

Comment:
Bikeshed: `unfreeze_values` sounds like an impure function to me. How about something like: `craft_unfrozen_copy()`?